File: src/test/kotlin/com/kdu/common/util/SSLConnectTest.java
package com.kdu.common.util;

import org.jsoup.nodes.Document;
import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

class SSLConnectTest {

    @Test
    void 정상적으로_documnet를_가져온다() throws Exception {
        Document document = SSLConnect.getDocument("C08");
        assertThat(document).isNotNull();
    }
}

File: src/test/kotlin/com/kdu/common/util/SSLConnectTest.kt
package com.kdu.common.util

import org.assertj.core.api.Assertions
import org.junit.jupiter.api.Test

internal class SSLConnectTest {

    @Test
    internal fun `정상적으로 Document를 가져온다`() {
        val document = SSLConnect.getDocument("C08")
        Assertions.assertThat(document).isNotNull
    }
}